client/components/CandidateReview/CandidateTestPlanRun/index.jsx

```diff
@@ -654,21 +673,8 @@ const CandidateTestPlanRun = () => {
           handleAction={submitApproval}
           handleHide={() => setFeedbackModalShowing(false)}
         />
-      ) : (
-        <></>
-      )}
-      {thankYouModalShowing ? (
-        <ThankYouModal
-          show={true}
-          handleAction={async () => {
-            setThankYouModalShowing(false);
-            navigate('/candidate-review');
-          }}
-          githubUrl={generalFeedbackUrl}
-        />
-      ) : (
-        <></>
       )}
+      {confirmationModal && confirmationModal}
```

Comment:
This is technically unnecessary but I thought it helped convey the possible of a no render for the modal.

Reply:
Yea, thinking you don't have to make conveying this clear but `{!!confirmationModal && confirmationModal}` could also display that?